# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rcs/hotfix.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- rcs-5.8.2/src/base.h.vanilla	2020-02-19 09:49:42.347776993 +0000
+++ rcs-5.8.2/src/base.h	2020-02-19 09:50:17.015776813 +0000
@@ -40,7 +40,7 @@
 #include <vfork.h>
 #endif
 
-#define exiting  _Noreturn
+#define exiting  // _Noreturn
 
 /* GCC attributes  */
 
